app/utils/config.js

```diff
@@ -35,6 +35,21 @@ export default EmberObject.create({
   decisionDocumentTypeId: '2b73f8e2-b1f8-4cbd-927f-30c91759f08b',
   minuteDocumentTypeId: 'e149294e-a8b8-4c11-83ac-6d4c417b079b',
   remarkId: '305E9678-8106-4C14-9BD6-60AE2032D794',
+  mail: {
+    defaultFromAddress: 'noreply@vlaanderen.be',
+    translationRequest: {
+      content: 'Beste,\n\nIn bijlage \n\n%%attachments%%\n\nGraag snel oplossen.',
+      subject: '[%%kaleidosenvironment] Aanvraag vertaling voor Publicatie %%nummer%%',
+    },
+    publishPreviewRequest: {
+      content: 'Beste,\n\nIn bijlage \n\n%%attachments%%\n\nVoor drukproef.',
+      subject: '[%%kaleidosenvironment] Aanvraag voor drukproef Publicatie %%nummer%%',
+    },
+    publishRequest: {
+      content: 'Beste,\n\nIn bijlage \n\n%%attachments%%\n\nVoor publicatie.',
+      subject: '[%%kaleidosenvironment] Aanvraag voor Publicatie %%nummer%%',
+    },
```

Comment:
Misschien heel nitpicky maar waarom placholders met %% en [%%] ipv bv {} zoals andere placeholders? Is er een specifieke reden dat dit moet voor bijvoorbeeld de mail integratie? Ik zie [%%kaleidosenvironemt] bv niet gereplaced worden hier waardoor ik verward ben.

Reply:
terecht dat je in de war bent. 
Ik ben ook in de war. Het is nog niet beslist wat de subject zal moeten worden. 
Ik heb hier gewoon een suggestie gedaan om iets met tokens in de subject te steken. 
Ik heb  hier met het percent symbool gewerkt omdat ik dat in andere systemen ook gebruikte maar dit kan uiteraard vanalles anders zijn.
De environment had ik er inderdaad nog niet in gestopt omdat het onderwerp ook nog fout is 
(nog niet beslist)
Hier moet zowiezo dus nog aan aangepast worden.

Reply:
Ik dacht gewoon al wat voorbereidend werk te doen. 
Maar dit zit niet in de sprint dus ik kan de PR wel weer sluiten als we het volgens het boekje doen.

Reply:
Euhm maakt mij niet uit.. ik weet dat er discussie was over of het by default ingevuld moest zijn met een specifieke content of niet.. da's echter niet onze call om te maken..

Voor mij mag het gewoon door als het een voorbereiding is op een volgend stuk.. Die symbolen maakt me niet uit, dat is maar wat flair die ertussen zit.. eender wat gaat daar werken dus dat is zeker geen blocker. Ik had het gewoon nog niet zien voorkomen :)

Reply:
@BenjaminDendas aanpassen naar {{}} lijkt me risicovol want dan mogen we ook geen spaces er in steken en lijkt het te werken als handlebars maar dat gaat het dan niet doen.
Op termijn moet dit ook configureerbaar zijn in een interface. Ik wou maar gewoon een sensible default geven.

Reply:
Dit zal op termijn ook editeerbaar moeten worden in een interface btw.